scalac-scoverage-plugin/src/main/scala/scoverage/plugin.scala

```diff
@@ -225,6 +238,11 @@ class ScoverageInstrumentationComponent(val global: Global, extraAfterPhase: Opt
       }
     }
 
+    def isUndefinedParameterInScalaJs(symbol: Symbol): Boolean = {
+      isScalaJsEnabled && symbol != null && symbol.isSynthetic && symbol.isMethod &&
+        symbol.nameString.contains("$default$") &&
+        symbol.tpe.resultType.annotations.exists(_.symbol.nameString == "uncheckedVariance")
```

Comment:
This is too broad, and maybe also too restrictive (e.g., I don't know what the `uncheckedVariance` is doing here).

Here is how we identify an undefined parameter in the real Scala.js compiler:
https://github.com/scala-js/scala-js/blob/4619d906baef7feb5d0b6d555d5b33044669434e/compiler/src/main/scala/org/scalajs/nscplugin/GenJSCode.scala#L2696-L2721
You should basically copy-paste that logic, along with the helper methods that it calls.

Reply:
Alternatively, you could disable *all* default parameters under Scala.js. It is less precise, but it will be much easier. In that case you should write
```scala
isScalaJsEnabled && symbol.hasFlag(scala.reflect.internal.Flags.DEFAULTPARAM)
```

Reply:
Copy-pasted from scala-js in https://github.com/scoverage/scalac-scoverage-plugin/pull/281/commits/f83f2ec8a438f9d781780cb31a27732922126164